data/toc.yaml

```diff
@@ -1156,6 +1156,8 @@ Manuals:
           section:
           - path: /desktop/hardened-desktop/enhanced-container-isolation/
             title: What is Enhanced Container Isolation?
+          - path: /desktop/hardened-desktop/enhanced-container-isolation/config/
+            title: Configuration
```

Comment:
I prefer it the way it is because the "What is ECI?" section provides a good enough background, and then the config section describes how to configure the feature. Then the "How does it work" and "Key features and benefits" sections provide a deeper dive.

Reply:
> I also wonder if we call the page 'Advanced configuration (Beta)'

That sounds good, thanks!